components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -43,13 +50,96 @@ std::string GenerateMnemonicInternal(uint8_t* entropy, size_t size) {
   wally_free_string(words);
   return result;
 }
+
+std::string GetInfuraProjectID() {
+  std::string project_id(BRAVE_INFURA_PROJECT_ID);
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  if (env->HasVar("BRAVE_INFURA_PROJECT_ID")) {
+    env->GetVar("BRAVE_INFURA_PROJECT_ID", &project_id);
+  }
+  return project_id;
+}
+
+bool GetUseStagingInfuraEndpoint() {
+  std::string project_id(BRAVE_INFURA_PROJECT_ID);
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  return env->HasVar("BRAVE_INFURA_STAGING");
+}
+
+// Precompiled networks available in native wallet.
+const brave_wallet::mojom::KnownNetwork kKnownNetworks[] = {
+    {"0x1", "Ethereum Mainnet", "mainnet", "https://etherscan.io"},
+    {"0x4", "Rinkeby Test Network", "rinkeby", "https://rinkeby.etherscan.io"},
+    {"0x3", "Ropsten Test Network", "ropsten", "https://ropsten.etherscan.io"},
+    {"0x5", "Goerli Test Network", "ropsten", "https://goerli.etherscan.io"},
+    {"0x2a", "Kovan Test Network", "kovan", "https://kovan.etherscan.io"},
```

Comment:
For these, can we also please return 
```nativeCurrency: { symbol: 'ETH', name: 'Ethereum', decimals: 18 }```
This will be needed for the `Confirm Transaction` panels.

Reply:
done

Reply:
Looks good!